### DISCUSSION TITLE
Do you have any alternative company standards for child growth/development charts vs what we got from Roche standards team below?

### DISCUSSION THREAD

**Discussion by rossfarrugia** [pharmaverse/admiralpeds#6] [General]
![image](https://github.com/pharmaverse/admiralpeds/assets/82581364/d1db05a4-a1a7-4e4c-9f69-e04edc9c36cf)

ALSO: do you have any other areas that you'd like to as priority cover in the first release of admiralpeds?

**Comment by Slu510** [comment]
Will the team also be incorporating head circumference (HC) data? HC is another parameter used in clinical medicine to ensure proper growth as well as monitoring for micro- or macrocephaly.

**Reply by CDC-DNPAO** [reply]
Head circumference is in WHO reference data, and the code is in my whoanthro package. I think it would be easy to include this info in the new package.

**Reply by Slu510** [reply]
Do you also have CDC HC values as well? Full disclosure: I'm not sure what sponsors would prefer but if CDC data is readily available, it might be something to consider adding to the package.

**Reply by CDC-DNPAO** [reply]
CDC decided in ~2007 to use WHO reference data for kids under 2 y.  Many CDCers were upset at this decision, but CDC didn't have good data from national surveys for kids < 2 y of age.  I think that is WHO data is being used for infant wt, ht, BMI (or wt-for-length), it makes sense to also use WHO data for head circumference.  Another problem with the CDC infant data is that the LMS values are given only for each month of age, so interpolation is needed.  In contrast, the WHO reference data is for each day of age.

**Reply by Slu510** [reply]
Oh interesting! Didn't not know that.

**Reply by rossfarrugia** [reply]
UPDATE: Head circumference was included in package release 0.1.0

**Comment by CDC-DNPAO** [comment]
The other 'big' growth standard used in many analyses is the IOTF (Intl Obesity Task Force) standards.  See 
http://www.ncbi.nlm.nih.gov/pubmed/22715120
The reference data for these charts is based on children from 6 countries (UK, US, Netherlands, Singapore, etc).  Like the CDC and WHO charts, these charts are also based on LMS parameters.

**Reply by rossfarrugia** [reply]
UPDATE: in package release 0.1.0 we included a reference to IOTF standards to make clear any user could use these instead, but we decided not to add this metadata itself as part of the package.

**Comment by kabis-ops** [comment]
Hi all,
Not sure it fits in this issue but the Growth velocity analysis is another very common analysis for pediatric patients: it allows to check how much a kid grow during a certain period of time. 

Growth velocity (cm/year) = (height (cm) at time t2 -height (cm) at time t1)/time between measurement (years). 

We usually summmarize this information for e.g. the First 6 months on study, First 12 months, or First x months after an event, .....and  plot it against the growth velocity derived from the WHO/CDC values to identify trend/outliers.

**Reply by rossfarrugia** [reply]
@kabis-ops i'm thinking we put this one on the backlog for a future release, so we can focus on the current outline we have for our first 0.1.0 release as there's already quite bit to do. will make an issue later for this.

**Comment by CDC-DNPAO** [comment]
All (or almost all) investigators think it's inappropriate to use the
WHO/CDC *cross-sectional* charts to examine growth velocity, but I don't
know if everyone understands that this is problematic.

WHO published growth velocity standards in 2017, but I'm unsure if these
are widely used and if they'd apply only to the relatively affluent,
breastfed kids examined by WHO.  The data/charts are available for weight
velocity,  height velocity, and head circumference velocity at Weight
velocity (who.int)
<https://www.who.int/tools/child-growth-standards/standards/weight-velocity>
etc.
 The 'expected' increases depend on the current value and the time interval
between measurements.

Here's a letter by Zemel that points out the problems of using the
cross-sectional charts for a child's velocity A commentary on the
construction of weight velocity charts - PubMed (nih.gov)
<https://pubmed.ncbi.nlm.nih.gov/19841252/>

**Reply by rossfarrugia** [reply]
@CDC-DNPAO @kabis-ops any input from your sides on whether we should include something for growth velocity in our release 0.2.0 of admiralpeds? from David's reply above i'm unclear as to whether there is value if the underlying charts themselves are inappropriate or under-used. FYI @Fanny-Gautier @zdz2101

**Reply by rossfarrugia** [reply]
FYI @Fanny-Gautier @zdz2101 that David replied saying he doesn't know so much on these standards so can't comment more. Let's wait and see if we hear any other demand for this.
